.github/workflows/ci.yml

```diff
@@ -31,4 +34,4 @@ jobs:
         with:
           use_server_key: true
           role_arn: ${{ vars.AWS_ROLE_ARN }}
-          command: python test.py
+          command: CI=1 python test.py
```

Comment:
test.py is removed with this PR. Probably need to change this line?

Reply:
Yes, good call. Fixed.